src/Artifacts.jl

```diff
@@ -0,0 +1,589 @@
+module Artifacts
+
+import Base: get, SHA1
+import ..depots1
+import ..GitTools: tree_hash, set_readonly
+using ..BinaryPlatforms
+import ..TOML
+import ..Types: parse_toml
+import ..PlatformEngines: download_verify_unpack, probe_platform_engines!, package
+
+export create_artifact, artifact_exists, artifact_path, remove_artifact, verify_artifact,
```

Comment:
Are all the functions in here public (i.e. we want to show them in the docs).

Reply:
I see no reason not to.